samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -95,6 +95,7 @@ Button 1:
    .. note::
        The advertising is disabled while the Fast Pair Provider is connected to a Bluetooth Central.
        The discoverable advertising is automatically switched to the not discoverable advertising with the show UI indication mode after a 10-minute timeout.
+       The advertising mode is automatically switched to the not discoverable advertising with the show UI indication mode after a Bluetooth Central successfully pairs.
```

Comment:
New proposal:
```suggestion
   .. note::
       The Bluetooth advertising is active only until the Fast Pair Provider connects to a Bluetooth Central.
       After the connection, you can still switch the advertising modes, but the switch will come into effect only after disconnection.
       The discoverable advertising is automatically switched to the not discoverable advertising with the show UI indication mode in the following cases:
       
       * After a 10-minute timeout.
       * After a Bluetooth Central successfully pairs.
```
@alstrzebonski , @MarekPieta , would this work?

Reply:
For me it's ok.

Reply:
Or maybe `* After 10 minutes of active advertising`? That would be more accurate.